src/main/kotlin/JobState.kt

```diff
@@ -0,0 +1,64 @@
+package franz.internal
```

Comment:
Oh, I only found _franz.internal_ so I used that too, but it did felt wrong to do so. I'll fix that.